recipes/UrbanSound8k/README.md

```diff
@@ -54,7 +54,41 @@ python train.py hparams/train_ecapa_tdnn.yaml --data_folder=/localscratch/UrbanS
 python train.py hparams/train_ecapa_tdnn.yaml --data_folder=/localscratch/UrbanSound8K/ --train_fold_nums=[1, 2, 3, 4, 5, 6, 7, 8, 9] --valid_fold_nums=[10] --test_fold_nums=[10] --output_folder=./results/urban_sound/fold_10
 `
 
-Note that the results for 10-fold must be compiled from the output folders and averaged manually.
+Note that the results for 10-fold must be compiled from the output folders and averaged using ./SoundClassification/harry_plotter.py (which computes average and displays confusions matrix using seaborn) or manually.
+
+# Perfomance without data augmentation (10 folds)
+
+* Results averaged using [harry_plotter.py](https://github.com/ZiRaymondal/speechbrain/tree/develop/recipes/UrbanSound8k/SoundClassification/harry_plotter.py)
+
+|**Label\Precision** |**Speechbrain**|
+|--------------------|-----------------------|
+|**dog_bark**        |  0.8133
+|**children_playing**|  0.7880          
+|**air_conditioner** |  0.7783
+|**street_music**    |  0.5960
+|**gun_shot**      |  0.7890           
+|**siren**         |  0.8759  
+|**engine_idling** |  0.6747              
+|**jackhammer**    |  0.5452
+|**drilling**      |  0.6761   
+|**car_horn**      |  0.7184
+|**Overall Mean Acc** |**0.72559**
+
+![Averaged confusion matrix](https://gitlab.com/Raymondaud.Q/lia-alternance-m2/-/raw/master/SpeechBrain-UrbanSound-Recipe/10_Fold/SANS_DATA_AUGMENTATION/Sum_Confu_SB_Default.png)  
```

Comment:
Same as above, this should be a relative link within the same repo, not a link to your external repository (so the image should be uploaded somewhere in your fork's branch, in whatever asset directory may be appropriate).